meta-dts-distro/recipes-dts/dts-scripts/dts-scripts_git.bb

```diff
@@ -0,0 +1,35 @@
+SUMMARY = "Dasharo Tools Suite scripts."
+SECTION = "tools"
+HOMEPAGE = "https://github.com/Dasharo/dts-v2"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=575ab8837c52eafc7b9eb8b244072bc7"
+
+SRCREV = "f8afaca3873821d13a9db79c944302d7ff9df259"
+SRC_URI = "git://github.com/Dasharo/dts-v2;protocol=https;branch=main"
+
+PV = "0.1+git${SRCPV}"
+
+S = "${WORKDIR}/git"
+
+RDEPENDS:${PN} = " \
+    packagegroup-coreboot-utils \
+    flashrom \
+    iasl \
+    tar \
+    i2c-tools \
+    i2c-tools-misc \
+    bash \
+    dmidecode \
+    acpi-call-dkms \
+    iotools \
+"
+
+do_configure[noexec] = "1"
+do_compile[noexec] = "1"
+
+do_install () {
+    install -d ${D}${sbindir}
+    install -d ${D}${sysconfdir}/profile.d
+    oe_runmake install DESTDIR="${D}${sbindir}" SYSCONFDIR="${D}${sysconfdir}/profile.d"
```

Comment:
DESTDIR should be just ${D}, I have sent you some examples, like here: https://github.com/Dasharo/meta-dts/pull/115#discussion_r1557371635

It is not the standard way of doing things.

Reply:
Please try with updated Makefile from the linked comment

Reply:
Verified, works fine, thanks.